packages/mermaid/src/diagrams/block/blockRenderer.ts

```diff
@@ -0,0 +1,120 @@
+import type { Diagram } from '../../Diagram.js';
+import * as configApi from '../../config.js';
+import { calculateBlockSizes, insertBlocks, insertEdges } from './renderHelpers.js';
+import { layout } from './layout.js';
+import { setupGraphViewbox } from '../../setupGraphViewbox.js';
+import insertMarkers from '../../dagre-wrapper/markers.js';
+import {
+  select as d3select,
+  scaleOrdinal as d3scaleOrdinal,
+  schemeTableau10 as d3schemeTableau10,
+} from 'd3';
+import { log } from '../../logger.js';
+
+import type { BlockDB } from './blockDB.js';
+import type { Block } from './blockTypes.js';
```

Comment:
```suggestion
import { log } from '../../logger.js';
import type { BlockDB } from './blockDB.js';
import type { Block } from './blockTypes.js';
```

Avoid empty lines in imports. The auto sort function will change everything if someone removes the space later.